### PR TITLE
Support mermaid workflow

### PR DESCRIPTION
**Pull request by Ja7ad**
Support mermaid for workflow

image

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -11,8 +11,19 @@ repository: pactus-project/PIPs
 
 plugins:
   - jekyll-sitemap
+  - jekyll-spaceship
```
Comment (b00f) [inline _config.yml:14]: Do we need to add another plugin, can we just  add `mermaid` plugin??

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ jobs:
       #     root: _site
 
       - name: HTML Proofer
-        run: bundle exec htmlproofer ./_site
+        run: bundle exec htmlproofer --ignore-urls "/mermaid.ink/" ./_site
```
Comment (b00f) [inline .github/workflows/ci.yml:61]: Why do we need to ignore them?

File: _config.yml (hunk 1 above)
Comment (Ja7ad) [reply _config.yml:14]: We need just mermaid

File: .github/workflows/ci.yml (hunk 2 above)
Comment (Ja7ad) [reply .github/workflows/ci.yml:61]: Htmlproof has an issue with the alt image, in the mermaid we can't set alt for svg.

File: PIPs/pip-1.md [hunk 3]
```diff
@@ -36,7 +36,24 @@ Once a PIP is approved, the implementation of the PIP can begin.
...
-![Status Workflow](../assets/pip-1/pactus-pip-workflow.png)
+```mermaid!
+flowchart TD
+    Draft --> Accepted
+    Draft --> Rejected
+    Draft --> Withdrawn
+    Draft --> Deferred
+    Accepted --> Final
+    Final -.-> Replaced
+    Accepted --> Active
```
Comment (b00f) [inline PIPs/pip-1.md:47]: Active is standalone state.

**Pull request merged**